IntAct.pm

```diff
@@ -140,18 +134,12 @@ sub new {
 
   $self->{feature_ac} = 1 if defined $param_hash->{feature_ac};   
   $self->{feature_short_label} = 1 if defined $param_hash->{feature_short_label};
-  $self->{feature_range} = 1 if defined $param_hash->{feature_range};
-  $self->{original_sequence} = 1 if defined $param_hash->{original_sequence};
-  $self->{resulting_sequence} = 1 if defined $param_hash->{resulting_sequence};
   $self->{feature_type} = 1 if defined $param_hash->{feature_type};
   $self->{feature_annotation} = 1 if defined $param_hash->{feature_annotation};
   $self->{ap_ac} = 1 if defined $param_hash->{ap_ac};
-  $self->{ap_symbol} = 1 if defined $param_hash->{ap_symbol};
-  $self->{ap_full_name} = 1 if defined $param_hash->{ap_full_name};
   $self->{interaction_participants} = 1 if defined $param_hash->{interaction_participants};
   $self->{pmid} = 1 if defined $param_hash->{pmid};
-  $self->{figure_legend} = 1 if defined $param_hash->{figure_legend};
```

Comment:
The options removed are still defined in $valid_fields
https://github.com/Ensembl/VEP_plugins/blob/26aa498cd1d51f1317e861d47b7cfd001420ca1f/IntAct.pm#L99